BetSystem.Tests/Endpoint/SportEventEndpointTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using BetSystem.Contract;
+using BetSystem.Contract.BusinnessLogic;
+using BetSystem.Endpoint;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Moq;
+
+namespace BetSystem.Tests.Endpoint
+{
+    public class SportEventEndpointTests
```

Comment:
Also you have a lot of methods in service - I don't see tests for the rest of methods:
        void AddTeamToSportEvent(int id, IdRequestDto idRequestDto);
        List<TeamDto> GetTeamsFromSportEvent(int id);
        void DeleteById(int id);
        List<SportEventDto> GetAllEvents();
        SportEventDto GetById(int id);
        void PostSportEvent(SportEventDto sportEventDto);
        void AddBetToSportEvent(BetOnEventDto betOnEventDto);
        List<BetOnEventDto> GetBetsFromSportEvent(int id);
        void AddResultToSportEvent(int id, EventResultDto eventResultDto);
        List<EventResultDto> GetResultsFromSportEvent(int id);
        void PutSportEvent(int id, SportEventDto sportEventDto);

Reply:
Done